libs/utils/utils.js

```diff
@@ -896,6 +896,11 @@ async function checkForPageMods() {
 }
 
 async function loadPostLCP(config) {
+  const georouting = getMetadata('georouting') || config.geoRouting;
+  if (georouting === 'on') {
+    const { default: loadGeoRouting } = await import('../features/georoutingv2/georoutingv2.js');
+    await loadGeoRouting(config, createTag, getMetadata, loadBlock, loadStyle);
```

Comment:
I think this is great, but think it really only makes sense to `await` on mobile devices. On desktop we should be able to call martech a little faster. 

In general I think the best solution would actually be to do the following: 
If a user is on a mobile device & georoutingMetadata is active we call the logic to figure out if we'll show the georouting modal before even loading the first block. This call should be non-blocking so that we can load the first section still right away, but at the same time, the moment we get a response informing us that the georouting modal will be displayed we load the modal as quickly as we can and make it blocking. 

I think this makes sense to do because the georouting modal is displayed often and is more importantly often LCP on mobile. 
The price we pay is of course kicking off one more network request on mobile devices before LCP, so we'd have to do some testing on what this materially means
do we lose one point of LH on average but gain 5 points when georouting is displayed. Etc.

Reply:
@auniverseaway Thoughts on this?

Reply:
Created a ticket to look into this approach https://jira.corp.adobe.com/browse/MWPW-142218